tensilelite/MANIFEST.in

```diff
@@ -1,3 +1,5 @@
+recursive-include rocisa *.hpp *.cpp *.txt *.cmake
+recursive-exclude rocisa/build *.txt
```

Comment:
I assume that this exclude here is because you are creating the build directory in the source dir. If you stop doing that, you should remove this exclude.

Reply:
Pip install does create a build folder, add this in case the CMakeCache.txt is copied

Reply:
Ok.